data/devices/roccat-kone-aimo-remastered.device

```diff
@@ -0,0 +1,4 @@
+[Device]
+Name=Roccat Kone AIMO Remastered
+DeviceMatch=usb:1e7d:2e2c
+Driver=roccat
```

Comment:
`Driver=roccat-kone-aimo` for now, see below.

Reply:
Done, I just copied the other one. I thought it was intended.